src/App.vue

```diff
@@ -84,6 +84,14 @@
       :webChatUrl="window['webChatUrl']"
     />
 
+    <!-- Display the Genesys WebMessage for SP/GP registrations only -->
+    <GenesysWebMessage
+      v-if="getFilingType === FilingTypes.REGISTRATION"
+      :genesysURL="window['genesysUrl']"
+      :environmentKey="window['genesysEnv']"
+      :deploymentKey="window['genesysId']"
+    />
```

Comment:
Does the GenesysWebMessage component have to be initialized in App.vue?  I know it's not how App.vue is used at present, but typically App.vue would render the appropriate component from the routes and not much more.  Would it be possible to put this component in the appropriate view instead?

Reply:
Actually, it's not even a Vue component. We made it a Vue component so we could easily drop it on a page like this. Where would such a component (or similarly, a dialog) be mounted?

Reply:
Since we want to render the component for only registrations, I believe we can put this component in the Registration views. What is the real benefit for doing that though?

Reply:
^^ That's actually a decent idea 👍

Reply:
@severinbeauvais Shall I proceed with this change or keep things as is? What do you think? If I do end up putting this component in the Registration views, I have to put in 3 different files. I don't think that's a good idea.

Reply:
On the other hand, if we put the new component in the Registration view here, and in the App view in Namerequest UI, then it lacks consistency  :/

What's best practice here.

Reply:
That's also a good point 😟

Reply:
> Since we want to render the component for only registrations, I believe we can put this component in the Registration views. What is the real benefit for doing that though?

There's a couple of benefits.  First, the v-if can be removed and the component only rendered when needed. Second, it cleans up the code.

Reply:
> @severinbeauvais Shall I proceed with this change or keep things as is? What do you think? If I do end up putting this component in the Registration views, I have to put in 3 different files. I don't think that's a good idea.

Oh yeah, good point -- we don't a single "Registration" view.

I'm OK with having in App.vue, but Jonathan started this conversation so I'll leave it for him to resolve.

Reply:
I think I'm going to leave it as. This way, it is consistent with the Namerequest UI and also I don't have to import it over and over again.

Reply:
I agree let's leave it as is.